packages/opentelemetry-exporter-prometheus/src/export/types.ts

```diff
@@ -0,0 +1,38 @@
+/*!
```

Comment:
Maybe move `types.ts` under `src/` and remove `export/`.

Reply:
Yes. I didn't realize khan had it there.

Reply:
I put it under this folder because they are exported publicly and I had seen other packages do that. Is that not something we do when types are exported?